sdk/src/azure/iot/az_iot_provisioning_client.c

```diff
@@ -339,6 +343,41 @@ AZ_INLINE az_result _az_iot_provisioning_client_payload_registration_state_parse
   return AZ_OK;
 }
 
+AZ_NODISCARD az_result _iot_provisioning_client_parse_operation_status(
```

Comment:
nit: Keep the `_az` prefix
```suggestion
AZ_NODISCARD az_result _az_iot_provisioning_client_parse_operation_status(
```